PickAPic/PickAPic/ViewController.h

```diff
@@ -8,7 +8,10 @@
 
 #import <UIKit/UIKit.h>
 
-@interface ViewController : UIViewController
+@interface ViewController : UIViewController <UIImagePickerControllerDelegate, UINavigationControllerDelegate> 
+
+@property (strong, nonatomic) IBOutlet UIImageView *imageView;
```

Comment:
You should try to keep all `@property` in the `.m` unless another object needs to access that property. In this case, the only object accessing `imageView` is `ViewController`, so you should move this to the `@interface` in the `.m`.